### PR TITLE
[Docs] Contributing / Project / Overview section restructure

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR breaks out contributing into new sections.

Screen Shot 2021-05-23 at 12 16 22 PM

### CODE REVIEW

**Review by navendu-pottekkat — COMMENTED** [review]
@leecalcote [This page](https://deploy-preview-3010--meshery-docs.netlify.app/project/contributing-guide) should be under Meshery CLI and not on the general contributing guide. The stuff mentioned there are CLI centric.

**Comment by leecalcote** [comment]
Consolidated two independent pages into a single page.

File: docs/pages/project/contributing/contributing-cli.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+Command structure and command behavior should be designed in such a way that they are intuitive. Users should ideally be able to understand what a command is used for without having to extensively go through the documentation. For example, `mesheryctl pattern apply -f <pattern name>` requires no further clarification as it is evident that the command will apply the pattern specified.
+
+Consistency is key when designing intuitive interfaces. Although `mesheryctl perf run -f <performance profile name>` may sound more intuitive, users who are experienced in using the CLI will prefer the consistant verb `apply` over `run`. This will also ensure a consistent command language making memorizing easier.
+
+** Flags **
```
Comment (navendu-pottekkat) [inline docs/pages/project/contributing/contributing-cli.md:77]: Was this supposed to be bold?

**Review by navendu-pottekkat — APPROVED** [review]
Looks Good!

**Comment by leecalcote** [comment]
> Was \*quality\* intentional? 

Good eye. No, this is an accident. Fixing...

File: docs/pages/project/contributing/contributing-cli.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/project/contributing/contributing-cli.md:77]: Yes, this "Flags" is intentional. We need more sections with style "Commands" and "Subcommands".

File: docs/pages/project/contributing/contributing-cli.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+These guidelines are a collection of principles and conventions that need to be followed while designing mesheryctl commands. `mesheryctl` might be the interface that the users first have with Meshery. As such, `mesheryctl` needs to provide a great UX.
+
+The following principles should be taken in mind while designing `mesheryctl` commands-
+
+## Design Principles
+
+**1. Consistency is quality.**
+
+- _Consistency of interaction drives a *quality* user experience. Whether that experience is delightful or painful is a related, but separate consideration. Meshery's behavior of user interactions should be consistent even when their user experience is poor._
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-cli.md:57]:
```suggestion
- _Consistency of interaction drives a quality user experience. Whether that experience is delightful or painful is a related, but separate consideration. Meshery's behavior of user interactions should be consistent even when their user experience is poor._
```

File: docs/pages/project/contributing/contributing-cli.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/project/contributing/contributing-cli.md:77]: But, you're right that in the doc's current state, it's a little out of place.

**Pull request merged**